quinn/src/connection.rs

```diff
@@ -79,29 +79,47 @@ impl Connecting {
 
     /// Convert into a 0-RTT or 0.5-RTT connection at the cost of weakened security
     ///
-    /// Opens up the connection for use before the handshake finishes, allowing the API user to
-    /// send data with 0-RTT encryption if the necessary key material is available. This is useful
-    /// for reducing start-up latency by beginning transmission of application data without waiting
-    /// for the handshake's cryptographic security guarantees to be established.
+    /// Returns `Ok` immediately if the local endpoint is able to attempt 0/0.5-RTT conversion. If
+    /// so, the returned [`Connection`] can be used to open streams and write data to them without
```

Comment:
s/open streams and write data/send application data/

Streams are not the only form of application data supported in 0-RTT. Application datagrams may be supported, and other forms might be added in future extensions.